crates/orderbook/openapi.yml

```diff
@@ -443,6 +443,30 @@ paths:
                   $ref: "#/components/schemas/Order"
         400:
           description: Problem with parameters like limit being too large.
+  /api/v1/token/{token}/native_prices:
```

Comment:
maybe native_price without the s at the end?

Reply:
Fixed.